vmm/src/device_manager.rs

```diff
@@ -900,8 +904,20 @@ impl DeviceManager {
                     vm_virtio::Net::new_with_tap(tap, Some(&net_cfg.mac), net_cfg.iommu)
                         .map_err(DeviceManagerError::CreateVirtioNet)?
                 } else {
-                    vm_virtio::Net::new(net_cfg.ip, net_cfg.mask, Some(&net_cfg.mac), net_cfg.iommu)
-                        .map_err(DeviceManagerError::CreateVirtioNet)?
+                    vm_virtio::Net::new(
+                        net_cfg
+                            .ip
+                            .ok_or(())
+                            // FIXME
```

Comment:
@sameo @rbradford @sboeuf noob question, I did not find a nice way to propagate an string with the current error types could you give me an advice on how this would be improved ?